.travis.yml

```diff
@@ -35,6 +38,29 @@ after_success:
   - pip install codecov
   - codecov
 
+matrix:
+  allow_failures:
+    - os: osx
+  include:
+    - name: "osx + builtin python3"
+      os: osx
+      language: sh
```

Comment:
Language is python. why specify as shell?

Reply:
:D 

tldr: doesn't work with osx. 

Long version: when you specify `language: python` along with `os: osx`, it tries to download the specified python version from a location that doesn't have it and the build errors out. I was trying some complicated mess of an approach earlier on with homebrew (where I was facing some issues with cache-ing) but I realized early yesterday that the osx images on azure pipelines already comes with a version of `python3` (3.6.5), so I am using that directly now.

Reply:
> Long version: when you specify language: python along with os: osx, it tries to download the specified python version from a location that doesn't have it and the build errors out.

That sounds like a travis bug to me. Is there an upstream issue for that? If not, can you please create one? In either case, we need a link to that issue as a comment in the yml file.

Reply:
It's called out fairly clearly here: https://docs.travis-ci.com/user/languages/python/